applications/tari_stratum_transcoder/src/proxy.rs

```diff
@@ -480,6 +480,64 @@ impl InnerService {
         proxy::json_response(StatusCode::OK, &json_response)
     }
 
+    async fn handle_get_fee(
+        &self,
+        request: Request<json::Value>,
+    ) -> Result<Response<Body>, StratumTranscoderProxyError> {
+        let request = request.body();
+        let transactions = match request["params"]["transactions"].as_array() {
+            Some(v) => v,
+            None => {
+                return proxy::json_response(
+                    StatusCode::OK,
+                    &json_rpc::error_response(
+                        request["id"].as_i64(),
+                        1,
+                        "`transactions` field is empty or an invalid type for transfer request.",
+                        None,
+                    ),
+                )
+            },
+        };
+
+        let mut grpc_transaction_info = Vec::new();
```

Comment:
nit: just a note that this is a situation where a `Vec::with_capacity()` is a slight optimization